default.nix

```diff
@@ -46,6 +46,7 @@ let
 
   copyToRegistry = image: pkgs.writeShellScriptBin "copy-to-registry" ''
     echo "Copy to Docker registry image ${image.imageName}:${image.imageTag}"
+    ${skopeo-nix2container}/bin/skopeo --insecure-policy inspect docker://${image.imageName}:${image.imageTag} ||
```

Comment:
I'm not in favor of doing this because images with a tag `latest` [1] would be skipped, even if they differ.

[1] i know this is not a good pattern, but it is currently a common one in the container world...